### DISCUSSION TITLE
loss of Ex command auto-completion

### DISCUSSION THREAD

**Discussion by wulymammoth** [folke/noice.nvim#861] [Q&A]
Sometimes the command line doesn't work as expected anymore and I'm wondering if others face the same issue...

These no longer auto-complete some of the time:
- for example, typing `:La` and then tab should display a list of commands with this prefix, typically `Lazy` is listed first
- `:b` and tab should allow me to pan through my buffers

I use Ex commands pretty often, but gets borked at times.

System deets:
* NVIM v0.10.0
* Build type: Release
* LuaJIT 2.1.1713773202

**Comment by folke** [comment]
Never had that before.
